content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -210,6 +210,12 @@ In releases older than Debian 12 and Ubuntu 22.04, folder `/etc/apt/keyrings` do
    sudo apt-mark hold kubelet kubeadm kubectl
    ```
 
+5. (Optional) Enable the kubelet service before running kubeadm:
```

Comment:
Even better: give the reader a hint about how to decide whether to do this optional step.